### PR TITLE
carfield_fix: Skip OT boot if no secure boot

### PR DESCRIPTION
**Pull request by niwis**
Do not wait for OT to boot if `secure_boot` is deasserted. Lets TB terminate for test cases that do not initialise OT.

### CODE REVIEW

**Review by alex96295 — APPROVED** [review]
LGTM

Can we manually run the litmus with this PR before merging?

**Comment by niwis** [comment]
@alex96295 the litmus tests are running, and the tests are terminating now as expected (no more `Wait for OT to boot...`). Do you think we should wait for all of them to complete (~1day) or merge and continue development on master?

**Comment by alex96295** [comment]
> @alex96295 the litmus tests are running, and the tests are terminating now as expected (no more `Wait for OT to boot...`). Do you think we should wait for all of them to complete (~1day) or merge and continue development on master?

If you agree, I would pause merging for a day more and wait for the final results

**Pull request merged**
